.github/workflows/ci.yml

```diff
@@ -69,9 +92,27 @@ jobs:
         if:   always()
         run:  docker-compose logs
 
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
       - name: Run Integration Tests
         run: echo TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO
 
+  push-to-ecr:
```

Comment:
Can we rename this `build` please for consistency with the new CI/CD pipeline job names?

Reply:
Will do